sympy/geometry/tests/test_polygon.py

```diff
@@ -611,3 +613,43 @@ def test_cut_section():
     p3, p4 = p.cut_section(Line((0, 0), slope=0))
     assert p3 == Polygon(Point2D(0, 10), Point2D(0, 0), Point2D(20, 0), Point2D(20, 10))
     assert p4 == None
+
+    # case where the line does not intersect with a polygon at all
+    raises(ValueError, lambda: p.cut_section(Line((0, a), slope=0)))
+
+def test_type_of_triangle():
+    # Isoceles triangle
+    p1 = Polygon(Point(0,0), Point(5,0), Point(2,4))
+    assert p1.is_isosceles() == True
```

Comment:
Please have a look at the PEP8 style and add spacing aroung the commas. For instance this should be `p1 = Polygon(Point(0, 0), Point(5,0), Point(2,4))`